sec/steenrod.tex

```diff
@@ -60,73 +122,48 @@ \section{Steenrod diagonals}
 	Thus, this tight coherent section defines a vertex $-x=(-y,y)$ of $D_P$.
 \end{proof}
 
-\begin{definition}
-	A \emph{Steenrod diagonal} on $P$ is the datum of a frame which possess the bot-top property with respect to $D_P$.
-\end{definition}
+\anibal{Furthermore, the action of reversing .... corresponds to ... and to ...}
+
+\subsection{Steenrod diagonals}
 
-Observe that a Steenrod diagonal of $P$ defines a Steenrod diagonal on every face of $P$. 
+%\begin{definition}
+%	A \emph{Steenrod diagonal} on $P$ is the datum of a frame which possess the bot-top property with respect to $D_P$.
+%\end{definition}
+%
+%Observe that a Steenrod diagonal of $P$ defines a Steenrod diagonal on every face of $P$.
+%\anibal{Why?}
 
-According to \cref{e:centrally-symmetric}, the iterated fiber polytope of $P$ is a family of centrally symmetric polytopes. 
-Moreover, \cref{e:zonotopes} shows that if $P$ is a zonotope, then the iterated fiber polytope of $P$ is a familty of centrally symmetric zonotopes. 
-\Guillaume{check:the product of zonotopes is a zonotope}
+%\anibal{Do you mean $D_P$ here?}
```

Comment:
It is true in both cases: If D_P is a zonotope, and if P is a zonotope, since in this case P x P is also a zonotope (just checked this), which implies that D_P is a zonotope